s/types.go

```diff
@@ -355,3 +357,37 @@ var rootSupervisorName = ""
 // childSepToken is the token use to separate sub-trees and child names in the
 // supervision tree
 const childSepToken = "/"
+
+type supervisionError struct {
+	supRuntimeName string
+	childErr       *c.ErrorToleranceReached
+	terminateErr   terminateError
+}
+
+func (err supervisionError) String() string {
```

Comment:
Can you note which child?

Reply:
Same ordeal as in a [previous comment](https://github.com/capatazlib/go-capataz/pull/30#discussion_r399932065) I made, skipping this suggestion for now